engine/gamesys/src/gamesys/resources/res_sound_data.cpp

```diff
@@ -18,6 +18,21 @@
 
 namespace dmGameSystem
 {
+
+    dmSound::SoundDataType TryToGetTypeFromBuffer(char* m_Buffer, dmSound::SoundDataType default_type)
+    {
+        dmSound::SoundDataType type = default_type;
+        if (m_Buffer[0] == 'O' && m_Buffer[1] == 'g' && m_Buffer[2] == 'g')
```

Comment:
positions according to format spec

Reply:
What happens if someone sets a bad data to it, e.g. the byuffer length is smaller than 11 bytes?

Reply:
Also, unless the function is used in a unit test, it can be `static`